GalaxyZooTouchTable/GalaxyZooTouchTable/Lib/CommonData.cs

```diff
@@ -0,0 +1,50 @@
+﻿using GalaxyZooTouchTable.Models;
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace GalaxyZooTouchTable.Lib
+{
+    public class CommonData : INotifyPropertyChanged
```

Comment:
The name and purpose of this class is kind of confusing. Does it just contain all the user classes?

Could you change the name of the class or add a comment to better explain the purpose of the class.

Reply:
`GlobalData` might be a better title here.